File: Workspace/src/PrimeLs.java
import java.util.*;
import java.util.Scanner;
public class PrimeLs {  
    public static void main(String args[]){    
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the minimum: ");
        long x = input.nextLong();
        System.out.print("Enter the maximum: ");
        long y = input.nextLong();
        List<Long> list=new ArrayList<Long>();
        if (x <= 2) {
            list.add (2L);
        }
        while (x<=y) {
            for (long i = 2; i < x; i++) {
                if (x % i == 0) {
                    break;
                }
                if (i==x-1) {
                    list.add (x);
                }
            }
            x++;
        }
        for (Long primenumber:list) {
            System.out.printf("%d\n",primenumber);
        }  
        System.out.printf("The length of the list is %d\n", list.size());
    }  
}

File: Workspace/src/PrimeLs.py
print 'Enter the minimum: '
x = input()
print('Enter the maximum: ')
y = input()
primelist = []
if x <= 2:
    primelist.append (2)
while x<=y: 
    i=2
    while i < x:
        if x % i == 0:
            break
        if (i==x-1):
            primelist.append(x)
        i=i+1
    x=x+1
for s in primelist:
    print s
print 'The length of the list is ', len(primelist)
